packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -456,8 +456,11 @@ async function getMultiEnvFolders(projectPath: string): Promise<any> {
 
 async function backup(projectPath: string): Promise<void> {
   const fx = path.join(projectPath, `.${ConfigFolderName}`);
-  const backup = path.join(fx, "migrationbackup");
-  await fs.ensureDir(backup);
+  const backup = path.join(projectPath, ".backup");
```

Comment:
What if there is an existing `.backup` folder under the project root, as the directory name is quite common?